translations/Readme.md

```diff
@@ -0,0 +1,45 @@
+# About Translation
+
+## How to use the translation utilities
+
+### On Windows
+
+#### 1. Adding translation `(.ts)` files
+
+1. Open `lupdate.cmd` in editor. (e.g, VSCode or Notepad, etc.)
+2. Append your new file name to the end of the first line, in the format **locale_**_language-code_**.ts** (e.g, locale_de, local_zh-cn, etc.).
+3. Run the command file in cmd.
+4. You will find the new file in the same location as the `lupdate.cmd`.
+
+#### 2. Translate the strings
+
+Run `Qt Linguist.exe` and import .ts files.
+
+#### 3. Generating / Updating `.qm` files
+
+1. Just double click on `lrelease.cmd` or run it in `cmd.exe`.
+2. Run the command file in cmd.
+3. You will find the new file in the same location as the `lrelease.cmd`.
+
+#### 4. Correct Your Translation
+
+Copy and move the whole `batch_export` folder to  MuseScore `plugin` (You may check the path in MuseScore.exe, from the `top menu` &rarr; `File` &rarr; `Preferences...` &gt; `General` tab &gt; `Folders` section) to test the translation.
+
+### On Linux / macOS
+
+Use the `translation_utility.sh`.
+
+This is a 2-in-1 utility, which enables you to process `.ts` and `.qm` files in an efficient way.
+
+Before running the script, you have to assign permissions to `translation_utility.sh` using `sudo chmod u+x`.
+
+Then you can double click on it or run it in a shell. The instructions are shown in the UI.
+
+#### Correct Your Translation
+
+Copy and move the whole `batch_export` folder to  MuseScore `plugin` (You may check the path in MuseScore.app, from the `top menu` &rarr; `MuseScore` &rarr; `Preferences...` &gt; `General` tab &gt; `Folders` section) to test the translation.
```

Comment:
That's not to correct, but to control the translations. And not needed at all, if you're running all these steps from the 'correct' directory. The translation step itself is still missing though

Reply:
I updated it with "Preview"

Reply:
Now, if `.ts` files are generated successfully, the `Linguist.app` will automatically launch.